styles/class.module.css

```diff
@@ -0,0 +1,60 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  align-items: left;
+  flex-direction: column;
+  padding: 1rem 1.5rem;
+}
+
+.backbutton {
+  width: fit-content;
+}
+
+.navmenu {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: flex-start;
+  list-style-type: none;
+  height: 100%;
+  white-space: nowrap;
+  padding: 1rem;
+}
+
+.navtitle {
+  composes: headlineLarge from "./globals.css";
+  width: 100%;
+  padding-bottom: 2rem;
+  padding-left: 6px;
+  position: relative;
+}
+
+.navitem {
+  margin-bottom: 1rem;
+  width: 100%;
+  position: relative;
+}
+
+.navlink {
+  composes: labelLarge from "./globals.css";
+  text-decoration: none;
+  position: relative;
+  padding-left: 6px;
+  border-left: 3px solid transparent;
+}
+
+.clicked {
+  composes: labelLarge from "./globals.css";
+  padding-left: 6px;
+  border-left: 3px solid var(--orange);
+}
+
```

Comment:
the bug i pointed out with the navbar might also be because of this? Take a look